packages/opds-web-client/src/components/BasicAuthForm.tsx

```diff
@@ -60,14 +60,20 @@ export default class BasicAuthForm extends React.Component<BasicAuthFormProps, B
 
   validate() {
     let login = (this.refs["login"] as any).value;
+    let password = (this.refs["password"] as any).value;
 
     if (!login) {
       this.setState({
         error: `${this.loginLabel()} is required`
       });
       return false;
+    } else if (!password) {
+      this.setState({
+        error: `${this.loginLabel()} is required`
```

Comment:
There are some libraries where a password is not required. I think there's something in the auth document that indicates that.

Reply:
Gotcha. I'll write a comment for that because it confused me.